crates/vcs/src/lib.rs

```diff
@@ -4,8 +4,48 @@ mod loader;
 mod svn;
 mod vcs;
 
+use std::{collections::HashSet, path::Path};
+use strum::Display;
+
 pub use errors::VcsError;
 pub use git::Git;
 pub use loader::*;
 pub use svn::Svn;
 pub use vcs::*;
+
+#[derive(Debug, Display)]
+pub enum SupportedVcs {
```

Comment:
Let's use this enum instead of creating another https://github.com/moonrepo/moon/blob/master/crates/config/src/workspace/vcs.rs#L7